.travis.yml

```diff
@@ -79,6 +79,12 @@ jobs:
   - stage: Targets
     env: C=lm32.lite P=ice40_hx8k_b_evn F=stub T="base"
 
+  - stage: Targets
+    env: C=lm32.lite P=ice40_up5k_b_evn F=stub T="base"
+
+  - stage: Targets
+    env: C=lm32.minimal P=ice40_up5k_b_evn F=stub T="base"
```

Comment:
Why is there a `lm32.lite` and `lm32.minimal`?

Reply:
Because I'm on the fence about which version we should use; I _really_ want the cache of `lm32.lite`, but it makes the design more difficult to route. `lm32.minimal` is indeed easier to route, but lacks cache.

Reply:
Perhaps it might. I guess for now I'll disable `lm32.lite` from the build matrix, but I will be actively testing in locally.